### PR TITLE
5228 create control panel to communicate with each environments external api

### PR DESCRIPTION
**Pull request by zakiahmed-worldvision**

### CODE REVIEW

File: WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs [hunk 1]
```diff
@@ -94,6 +84,59 @@ public async Task<List<FeatureSwitchViewModel>> GetAllFeatureLists()
             return featureViewModel;
         }
 
+        /// <summary>
+        /// Update Feaure Flag
+        /// </summary>
+        /// <param name="flag"></param>
+        /// <param name="inputFeatureNames"></param>
+        /// <returns></returns>
+        public async Task<ActionResult> Update(bool flag, string inputFeatureNames)
```
Comment (geethakrishnamoorthy) [inline WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs:93]: - When I look at the MVC Page, We can able to update the N number of records based on the country sites and selected records.
- But in code, it seems to be a general update for all records in all country sites even though if we have selected records. 
- It's totally messy. You have to revamp this code completely for the update, reset, and all the methods.
- Before making the development code, draft the pseudo-code first and get the confirmation from your seniors and then write the code.

Comment (zakiahmed-worldvision) [reply WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs:93]: Yes thats right, it is a global update. A separate story - 5620 has been created to address this comment.

Comment (zakiahmed-worldvision) [reply WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs:93]: All other methods were code reviewed for story - 5229

**Review by geethakrishnamoorthy — CHANGES_REQUESTED** [review]
1) No Need for the project called BAL and DAL.
2) Move APIClient, Helper, Model, and APIClientFactory folder in WV.FeatureSwitch.Dashboard.Web project.
3) You did not configure serilog in your project, then how you logged your errors? Check the WVUnited4Admin application project.
4) Remove unnecessary files and Menu from project.
5) you no need to create Serilog.cs file. You can call directly serilog logger in your controller methods.. 
6) In FeatureSwitchServiceApiUrls.cs file, Should rename 'eventApiServicePrefix' as appropriate name 
7) Remove helperurls.cs file.
8) You should have negative cases for Getallfeaturemethods in FeatureSwitchControllerTest file.

File: WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs [hunk 2]
```diff
@@ -22,35 +22,37 @@ public class FeatureSwitchController : Controller
...
 
-        public FeatureSwitchController(IFeatureSwitchFactory featureSwitchFactory, ILogger<FeatureSwitchController> logger, IConfiguration configuration)
+        public FeatureSwitchController(IFeatureSwitchFactory featureSwitchFactory, ILogger<FeatureSwitchController> logger, IConfiguration configuration, IEnumerable<FeatureSwitchViewModel> testFeatureModelList)
         {
             _featureSwitchFactory = featureSwitchFactory;
             _logger = logger;
             response = new ApiResponse();
             _baseUrl = (AppConfigValues.ApiBaseUrl == null) ? configuration.GetSection("ApiConfig").GetSection("ApiBaseUrl").Value : AppConfigValues.ApiBaseUrl;
             _listOfCountries = (AppConfigValues.ApiCountry == null) ? configuration.GetSection("ApiConfig").GetSection("ApiCountry").Value : AppConfigValues.ApiCountry;
+            _featureSwitchViewModelList = testFeatureModelList.ToList();
```
[the remark this answers is not in the document]
Comment (zakiahmed-worldvision) [reply WV.FeatureSwitch.Dashboard.Web/Controllers/FeatureSwitchController.cs:34]: All of these comments will be addressed in the new story "5715 - Rework for Story 5228"

**Comment by geethakrishnamoorthy** [comment]
I have merged the branch 5228 changes into develop. My feedback points will be addressed in another new story #5716

**Pull request merged**